src/GapsInCareHelpers.ts

```diff
@@ -212,6 +213,13 @@ export function generateGuidanceResponses(
           } else {
             dataRequirement.dateFilter = [dateFilter];
           }
+        } else if (df.type === 'notnull') {
+          const valueFilter = generateDetailedValueFilter(df);
+          if (dataRequirement.extension) {
+            dataRequirement.extension.push(valueFilter);
+          } else {
+            dataRequirement.extension = [valueFilter];
+          }
```

Comment:
This whole chunk of code isn't covered by unit tests. Perhaps we could create a set of tests real quick.